src/testdir/test_string_interpolation.vim

```diff
@@ -0,0 +1,123 @@
+" Test for string-interpolations $'${x} ${y.z}'
+
+scriptencoding utf-8
+
+func Test_string_interpolation()
```

Comment:
This function is not needed, if you make all the static functions global.

Reply:
func Test_string_interpolation()
  call s:basic()
  call s:appendix()
  call s:illformed()
endfunc